tck/src/main/java/ee/jakarta/tck/concurrent/spec/ManagedScheduledExecutorService/resourcedef/ManagedScheduledExecutorDefinitionTests.java

```diff
@@ -35,15 +40,40 @@
 	@ArquillianResource(ManagedScheduledExecutorDefinitionServlet.class)
 	URL baseURL;
 	
+	@ArquillianResource(ManagedScheduledExecutorDefinitionOnEJBServlet.class)
+	URL ejbContextURL;
+	
 	@Deployment(name="ManagedScheduledExecutorDefinitionTests", testable=false)
-	public static WebArchive createDeployment() {
-		return ShrinkWrap.create(WebArchive.class)
-				.addPackages(false, ManagedScheduledExecutorDefinitionTests.class.getPackage(),
+	public static EnterpriseArchive createDeployment() {
+		
+		WebArchive war = ShrinkWrap.create(WebArchive.class, "ManagedScheduledExecutorDefinitionTests_web.war")
+				.addPackages(false,
 						getFrameworkPackage(), 
 						getContextPackage(),
 						getContextProvidersPackage())
-				.addClasses(ContextServiceDefinitionServlet.class)
+				.addClasses(
+						ReqBean.class,
+						ManagedScheduledExecutorDefinitionServlet.class,
+						ManagedScheduledExecutorDefinitionOnEJBServlet.class,
+						ContextServiceDefinitionServlet.class)
+//				.addAsWebInfResource(ContextPropagationTests.class.getPackage(), "web.xml", "web.xml")
 				.addAsServiceProvider(ThreadContextProvider.class.getName(), IntContextProvider.class.getName(), StringContextProvider.class.getName());
+		
+		JavaArchive jar = ShrinkWrap.create(JavaArchive.class, "ManagedScheduledExecutorDefinitionTests_ejb.jar")
+				.addPackages(false, getFrameworkPackage(), ManagedScheduledExecutorDefinitionTests.class.getPackage())
+				.deleteClasses(
+						ReqBean.class,
+						ManagedScheduledExecutorDefinitionServlet.class,
+						ManagedScheduledExecutorDefinitionOnEJBServlet.class)
+				.addClasses(
+						ContextServiceDefinitionInterface.class,
+						ContextServiceDefinitionBean.class)
+				.addAsManifestResource(ManagedScheduledExecutorDefinitionTests.class.getPackage(), "ejb-jar.xml", "ejb-jar.xml");
+//				TODO document how users can dynamically inject vendor specific deployment descriptors into this archive
```

Comment:
Can this TODO be removed?  Is there a reason why vendors should need to supply their own deployment descriptors?  As far as I can tell, everything in your tests is sticking to standards that should work everywhere.

Reply:
See Kyle's previous comment on this topic.